docs/_posts/2021-03-04-relinquishing-our-humanity.md

```diff
@@ -12,7 +12,9 @@ image: assets/images/articles/relinquishing_our_humanity.jpg
 
 *The inadvertent consequence of well-intentioned technology*
 
-![man with VR headset holding a phone and entangled in wires]({{page.image}})
+![woman in a tank top using a VR headset surrounded by computer monitors]({{page.image}})
+Photo by [cottonbro studio](https://www.pexels.com/photo/a-woman-in-a-tank-top-using-a-vr-headset-8721318/)
```

Comment:
Thanks for remembering to updating the alt text! To be better about using it consistently, I created an issue to standardize it in front matter (https://github.com/techedcollab/techedcollab.github.io/issues/6).